daemon/smartcard.h

```diff
@@ -69,8 +69,8 @@ int
 smartcard_update_token_state(container_t *container);
 
 int
-smartcard_container_start_handler(smartcard_t *smartcard, control_t *control,
-				  container_t *container, const char *passwd);
+smartcard_container_ctrl_handler(smartcard_t *smartcard, control_t *control, container_t *container,
+				 const char *passwd, container_ctrl_t container_ctrl);
 
```

Comment:
add a doxygen comment here plz.
this was missing in the first place, so since you now changed the api here,
it would be grate if you could amend this now.

Reply:
Done